notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    aiplatform.init(project=PROJECT_ID, location=REGION, staging_bucket=BUCKET_URI)
</pre><p>Duplicate with the code below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>done</p>